ui/src/pages/version/components/forms/components/transformer/components/table_inputs/OrdinalEncoderMapper.js

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useState } from "react";
+import { EuiButtonIcon, EuiFieldText, EuiFormRow } from "@elastic/eui";
+import { InMemoryTableForm, useOnChangeHandler } from "@gojek/mlp-ui";
+
+export const OrdinalEncoderMapper = ({
+  mappings,
+  onChangeHandler,
+  errors = {}
+}) => {
+  const { onChange } = useOnChangeHandler(onChangeHandler);
+
+  const [items, setItems] = useState([
+    ...Object.keys(mappings).map((v, idx) => ({
+      idx,
+      original: v,
+      transformed: mappings[v]
+    })),
+    { idx: Object.keys(mappings).length }
+  ]);
+
+  useEffect(
+    () => {
+      let newMappings = {};
+      items
+        .filter(item => item.original && item.original !== "")
+        .forEach(item => {
+          newMappings[item.original] = item.transformed;
+        });
+      onChange("mapping")(newMappings);
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [items]
+  );
+
+  const onDeleteItem = idx => () => {
+    items.splice(idx, 1);
+    setItems([...items.map((v, idx) => ({ ...v, idx }))]);
+  };
+
+  const getRowProps = item => {
+    const { idx } = item;
+    const isInvalid = !!errors[idx];
+    return {
+      className: isInvalid ? "euiTableRow--isInvalid" : "",
+      "data-test-subj": `row-${idx}`
```

Comment:
what is `data-test-sub`?

Reply:
I don't know either. It's copied from renameColumn.js
What should be the right string?